ports/stm32/boards/PYBD_SF2/mpconfigboard.h

```diff
@@ -149,6 +149,28 @@ extern struct _spi_bdev_t spi_bdev2;
 #define MICROPY_HW_I2S1             (1)
 #define MICROPY_HW_I2S2             (1)
 
+// optimal I2S PLL configurations for HSE frequency = 25 MHz and PLLM = 20
+#define I2S_PLL_CONFIG \
```

Comment:
It would be great to automatically generate this table based on HSE (see eg `boards/pllvalues.py` which does it for the main CPU clock).  Would that be difficult to do?

Otherwise, these tables would be better off in `machine_i2s.c` so they can be reused by all boards that have the same HSE and PLLM.  You could use `HSE_VALUE` and `MICROPY_HW_CLK_PLLM` macros to detect which table to use.  And just use the default I2S PLL values if a table matching HSE/PLLM could not be found.

Reply:
Thanks for reviewing this PR.

> It would be great to automatically generate this table based on HSE (see eg `boards/pllvalues.py` which does it for the main CPU clock). Would that be difficult to do?

It should be possible.   I created a python program to calculate the I2S PLL values, so some of the core code already exists.  https://github.com/miketeachman/toolbox/blob/main/stm32_calc_i2s_pll.py.  I'll take a look at `pllvalues.py` for an example implementation.

Reply:
@dpgeorge  I updated the PR to automatically generate PLL config table.

Reply:
Great, thank you!